cfep-03.md

```diff
@@ -0,0 +1,47 @@
+
+<table>
+<tr><td> Title </td><td> Providing manually uploaded builds on the conda-forge anaconda channel</td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Hadrien Mary &lt;hadrien.mary@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sep 27, 2016</td></tr>
+<tr><td> Updated </td><td> Oct 17, 2019</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Some packages, for example Qt and gcc, are very hard to build on the currently used CIs (CircleCI, Travis, AppVeyor, and AzurePipelines) due to time limitation.
+
+While this is a small set of packages we need a protocol to upload builds on conda-forge anaconda channel that are built off-CIs (local builds).
```

Comment:
comma after packages; "anaconda" seems redundant after "conda-forge" (or maybe missing "and"/"or"?)

Reply:
it is conda-forge anaconda.org channel but I guess we can simplify that by saying just conda-forge's channel at this point.